viite-UI/src/model/RoadAddressProjectCollection.js

```diff
@@ -240,11 +240,22 @@
         discontinuity: Number($('#roadAddressProject').find('#discontinuityDropdown')[0].value),
         roadEly: Number($('#roadAddressProject').find('#ely')[0].value),
         roadLinkSource: Number(_.first(changedLinks).roadLinkSource),
-        roadType: Number($('#roadAddressProject').find('#roadTypeDropDown')[0].value)
+        roadType: Number($('#roadAddressProject').find('#roadTypeDropDown')[0].value),
+        userDefinedEndAddressM: -1
```

Comment:
Why magic value -1?

Reply:
Honestly I thought it would be a way for backend to identify if there are, or not, user defined calibration points.